radio/src/targets/taranis/board.cpp

```diff
@@ -136,40 +136,43 @@ void boardInit()
 #endif
 #endif
 
-// Support for FS Led to indicate battery charge level
-#if defined(FUNCTION_SWITCHES)
+// If the radio was powered on by dual use USB, halt the boot process, let battery charge
+#if defined(PWR_BUTTON_PRESS) && !defined(USB_CHARGER)
```

Comment:
@3djc Final thoughts on this line... leave as it is,  maybe `#if defined(PWR_BUTTON_PRESS) && defined(MANUFACTURER_JUMPER)` or something else? I'm leaning towards the middle option, or some variation of it, and it being a problem for another time if it changes... this at least fixes the immediate bug. Dealers choice ;)

Reply:
I think it only applies to Jumper radio, so I would drop the pwr button and just use jumper. If/when this needs changing, we can always re assess the situation:)